router/network_router.go

```diff
@@ -239,12 +239,16 @@ func (router *NetworkRouter) InitialPeers(resume bool, peers []string) ([]string
 		return storedPeers, nil
 	}
 
+	log.Println("Launch detected - using supplied peer list:", peers)
+	return peers, nil
+}
+
+func (router *NetworkRouter) FinishInitiateConnections() error {
```

Comment:
Can we rename this `CreateRestartSentinel`? It doesn't really have anything to do with connection initiation